libraries/vue/package.json

```diff
@@ -14,11 +14,11 @@
     "karma-firefox-launcher": "2.1.2",
     "karma-sourcemap-loader": "0.3.8",
     "karma-webpack": "5.0.0",
-    "vue-template-compiler": "2.6.14",
+    "@vue/compiler-sfc": "^3.1.0",
```

Comment:
Nit: Is this dependency required?

Ref: https://www.npmjs.com/package/@vue/compiler-sfc#:~:text=vue/compiler%2Dsfc%20deep%20import